src/internal/_compose.js

```diff
@@ -21,7 +24,7 @@
  *      squareThenDouble(5); //≅ double(square(5)) => 50
  */
 module.exports = function _compose(f, g) {
-  return function() {
+  return curryN(g.length, function() {
```

Comment:
Are you sure you want to do it at this level rather than on the `compose.js` level. This will be significantly slower as you'll be wrapping it at each step of the way in a curry